docs/validation/primitive-type.md

```diff
@@ -29,12 +29,12 @@ import { Elysia, t } from 'elysia'
 
 new Elysia()
     .get('/', () => 'Hello World!', {
-        body: t.String()
```

Comment:
The text below doesn’t make sense if you change `body` to `response`.

You can however replace it with this snipped. which uses `post` instead of `get` and uses the actual `body`:

```ts
new Elysia()
  .post('/', ({ body }) => `Hello ${body}`, {
    body: t.String(),
  })
  .listen(3000);
```